sklearn/linear_model/_huber.py

```diff
@@ -205,6 +205,18 @@ class HuberRegressor(LinearModel, RegressorMixin, BaseEstimator):
         A boolean mask which is set to True where the samples are identified
         as outliers.
 
+    See Also
+    --------
+    RANSACRegressor : RANSAC (RANdom SAmple Consensus) algorithm.
+    TheilSenRegressor : Theil-Sen Estimator robust multivariate regression model.
```

Comment:
Could you add the SGDRegressor as well. This estimator can get several loss functions as the Huber loss as well.

Reply:
This comment has not been addressed yet.

Reply:
It should be added now